Package.swift

```diff
@@ -37,8 +36,7 @@ let package = Package(
                 "SwiftCBOR",
                 .product(name: "Crypto", package: "swift-crypto"),
                 .product(name: "_CryptoExtras", package: "swift-crypto"),
-                .product(name: "Logging", package: "swift-log"),
-                .product(name: "X509", package: "swift-certificates")
+                .product(name: "Logging", package: "swift-log")
```

Comment:
Super nit - can you add a trailing comma so it's clear there's only one line that's changed